app/src/main/java/com/example/mywatchface/mvp/presenter/WatchFacePresenter.kt

```diff
@@ -0,0 +1,111 @@
+package com.example.mywatchface.mvp.presenter
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.os.Bundle
+import android.support.wearable.watchface.WatchFaceService
+import android.view.SurfaceHolder
+import com.example.mywatchface.ALPHA
+import com.example.mywatchface.HOURS_ALPHA
+import com.example.mywatchface.HOURS_HAND_LENGTH
+import com.example.mywatchface.MIDDLE
+import com.example.mywatchface.MINUTES_ALPHA
+import com.example.mywatchface.MINUTES_HAND_LENGTH
+import com.example.mywatchface.SECONDS_ALPHA
+import com.example.mywatchface.SECONDS_HAND_LENGTH
+import com.example.mywatchface.ZERO_FLOAT
+import com.example.mywatchface.mvp.WatchFaceContract
+import java.util.Calendar
+import java.util.TimeZone
+
+class WatchFacePresenter(private val view: WatchFaceContract.WatchFaceView) :
+    WatchFaceContract.WatchFacePresenter() {
+
+    private var mMuteMode: Boolean = false
+    var onTapEnabled = false
+    var newColor = false
+    var mAmbient: Boolean = false
+
+    override fun init(background: Bitmap) {
+        view.mCalendar = Calendar.getInstance()
```

Comment:
you can use something like view.getCalendarInstance() if not, you will have some issues to test this

Reply:
done